website/snippets/_whitespace-control.md

```diff
@@ -0,0 +1,7 @@
+
+<expandable alt_header="📌 Tidy your macros with whitespace control">
```

Comment:
suggest removing the emoji. the red sorta clashes with the orange, at least to me. wdyt?

![Screenshot 2024-02-16 at 9 05 27 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/0fc323ea-c201-4078-a482-69827679a476)

Reply:
thanks Ly! i added an emoji bc i thought it blended too much with text and didn't want users to miss this as it's a good tip. i also think it reads better after the code example and not in the middle of the config explanation. 

i don't think expendables works well in this use case so have updated it to details so it stands out a bit more. 
![Screenshot 2024-02-19 at 10 43 36](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/d30c23ea-4216-474d-b94e-00e26eb68569)
![Screenshot 2024-02-19 at 10 53 19](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/88d6a820-6d62-4955-8c7a-4cfb39a838f6)